cfme/tests/cli/test_appliance_console.py

```diff
@@ -1601,3 +1601,30 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
     assert "Validated failed with error" in result[-1], "Database Configuration is not failed"
 
     ap_console_tail.validate(wait="60s")
+
+
+@pytest.mark.manual
+def test_appliance_console_logfile_config_reboot():
+    """
+        test to verify logfiles disk
+
+    Bugzilla:
+      1720223
+
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Appliance
+        caseimportance: high
+        caseposneg: negative
+        initialEstimate: 1/4h
+        testSteps:
+            1. get appliance with extra disk
+            2. set "logfile configuration"
+            3. set "Harden Appliance Using SCAP Configuration"
+            4. reboot appliance
```

Comment:
Looks like this could be a part of the `setup` and 'Execute `df -h` command and check log files for new disk' or something like that could be a testStep.

Reply:
added `df -h` to testSteps and  `reboot appliance` is not setup part, it is just one step of test case,